src/main/webapp/app/entities/drug/drug-update.tsx

```diff
@@ -133,12 +131,15 @@ export const DrugUpdate = (props: IDrugUpdateProps) => {
               </ValidatedField>
               <FormGroup>
                 <Label>Code</Label>
-                <NcitCodeSelect
-                  ncit={drugEntity.nciThesaurus}
-                  onChange={selectedOption => {
-                    setSelectedNcit(selectedOption ? parseNcitUniqId(selectedOption.value) : undefined);
-                  }}
-                />
+                {drugEntity.nciThesaurus && (
+                  <NcitCodeSelect
+                    isMulti={false}
+                    ncit={drugEntity.nciThesaurus}
+                    onChange={selectedOption => {
+                      setSelectedNcit(selectedOption ? parseNcitUniqId(selectedOption.value) : undefined);
+                    }}
+                  />
+                )}
```

Comment:
This changes the intended functionality. The page is also served when adding a new entity which we need to allow user to select ncit code.